support/Environments/wheeler_gcc.sh

```diff
@@ -4,7 +4,61 @@
 # See LICENSE.txt for details.
 
 spectre_setup_modules() {
-    echo "All modules on Wheeler are provided by the system"
+    if [ -z ${SPECTRE_HOME} ]; then
```

Comment:
you can delete everything you've added here. I've added an ARPACK module on wheeler. same in the clang config. Please confirm it works :)

Reply:
Thanks for installing Arpack on wheeler. I have removed the part that installed Arpack on wheeler and checked that the new version works.